content/10.methods.md

```diff
@@ -91,6 +91,11 @@ We used the estimated probabilities and did not convert to a hard group assignme
 For example, a query to <https://genderize.io> on January 26, 2020 for "Casey" returns a probability of male of .74 and a probability of female of 0.26, which we would add for an author with this first name.
 Because of the limitations of considering gender as a binary trait and inferring it from first names, we only consider predictions in aggregate and not as individual values for specific scientists.
 
+Of 411 ISCB honorees, genderize.io fails to provide gender predictions for two names.
+Of 34,005 corresponding authors, 45 have empty fore name field (i.e., missing metadata), 1,466 have fore name that is None once simplified (i.e., initials), and 1,578 have fore name for which genderize.io did not provide predictions.
+We note that approximately 52% of these NA predictions are hyphenated names, which is likely because they are more unique and thus are more difficult to find predictions for.
```

Comment:
```suggestion
52% of the names where genderize.io failed include hyphens, presumably since hyphenated names are more unique and therefore not represented in their training data.
```